ontology/uco/observable/observable.ttl

```diff
@@ -3449,10 +3694,35 @@ observable:IPNetmask
 		;
 	rdfs:subClassOf observable:ObservableObject ;
 	rdfs:label "IPNetmask"@en ;
-	rdfs:comment "An IP netmask is a 32-bit \"mask\" used to divide an IP address into subnets and specify the network's available hosts."@en ;
+	rdfs:comment "An IP netmask is a 32-bit 'mask' used to divide an IP address into subnets and specify the network's available hosts."@en ;
 	sh:targetClass observable:IPNetmask ;
 	.
 
+observable:IPhone
```

Comment:
Committee decision point - capital "I", or lowercase?

Reply:
Since it's considered a Class, and an (mnemonic) IRI as opposed to a name, follow the rules for IRI's and Classes, viz using the capital "I". In the comment and label fields, since these are human readable only, apply the usual name of the device, viz with a lowercase "i".

Reply:
When we have come to an agreement, consolidate this design decision by adding it to the Design Document.

Reply:
We'll go with uppercase.